include/secp256k1.h

```diff
@@ -40,6 +40,26 @@ extern "C" {
 #  define SECP256K1_ARG_NONNULL(_x)
 # endif
 
+# if defined _MSC_VER || defined __CYGWIN__
+#  define SECP256K1_HELPER_DLL_IMPORT __declspec(dllimport)
```

Comment:
@evoskuil, you are right. I've checked the Visual Studio 2013 and it generates better code when dllimport is used (as described in the article).